EXAMPLES.md

```diff
@@ -2,343 +2,392 @@
+    },
+    (err, result) => {
+        result.message.code; // http result code
+        result.message.body.arn; // Arn of user / account
+        result.message.body.canonicalID; // Canonical id of account
+        result.message.body.shortid; // id of account / user
+        result.message.body.email; // account / user email address
+        result.message.body.accountDisplayName; // Name of account
+        result.message.body.IAMDisplayName; // Name of user (or undefined)
+    });
+```
 
-{
-    "message": {
-        "code": 200,
-        "message": "Search successful",
-        "body": {
-            "isTruncated": true,
-            "accounts": [
-                {
-                    "arn": "arn:aws:iam::000140171645:/accountName1/",
-                    "id": "000140171645",
-                    "name": "accountName1",
-                    "createDate": "2016-04-06T23:19:08+02:00",
-                    "emailAddress": "09801321243867278@example.com"
-                },
-                {
-                    "arn": "arn:aws:iam::000238854835:/accountName2/",
-                    "id": "000238854835",
-                    "name": "accountName2",
-                    "createDate": "2016-04-08T15:12:42+02:00",
-                    "emailAddress": "4679125458933413account1@domain.com"
-                }
-            ],
-            "marker": "2"
-        }
-    }
-}
+### Get email addresses
+
+For this route, you don't need to set up the accessKeyId and secretKey of the
+client.
+The following example show how to get email addresses of accounts from their
+canonical ids :
+
+```js
+client.getEmailAddresses(['canonicalId1', 'canonicalId2'], {
+    reqUid: 'some-request-id', // Request id to trace request (optionnal)
+}, (err, result) => {
+    result.message.code; // http result code
+    // Email address for 'canonicalId1', Can also be 'WrongFormat' or 'NotFound'
+    result.message.body['canonicalId1'];
+});
 ```
 
-#### list-account-users
+### Get canonical ids
+
+For this route, you don't need to set up the accessKeyId and secretKey of
+the client.
+The following example show how to get canonical ids of accounts from their
+email addresses :
+
+```js
+client.getEmailAddresses(['email1', 'email2'], {
+    reqUid: 'some-request-id', // Request id to trace request in logs (optionnal)
+}, (err, result) => {
+    result.message.code; // http result code
+    // Canonical id for 'email1', Can also be 'WrongFormat' or 'NotFound'
+    result.message.body['email1'];
+});
+```
 
-```sh
-$ bin/vaultclient list-account-users --help
+## Https
 
-  Usage: list-account-users [options]
+To enable https for vaultclient cli, you have to pass the option ```--https```
+to vaultclient cli. In the case of self-signed certificates, you also need to
+use the option ```--cafile <certificate-authority-path>```. Another option is
+using the option ```--noCaVerification``` to disable ssl certificate validation.
```

Comment:
I'd tranform the https section into subsection 1.1 (inside "cli"). Otherwise, once the reader has arrived here s/he might have tried a couple of commands without understanding why they failed.